implementations/rust/ockam/ockam_node/src/storage/database/migrations/node_migrations/sql/postgres/20250115100000_create_database.sql

```diff
@@ -21,42 +21,46 @@ CREATE UNIQUE INDEX IF NOT EXISTS name_index ON _rust_migrations (name);
 CREATE TABLE identity
 (
     tenant_id      TEXT NOT NULL, -- Tenant identifier
-    identifier     TEXT NOT NULL UNIQUE,
-    change_history TEXT NOT NULL
+    identifier     TEXT NOT NULL,
+    change_history TEXT NOT NULL,
+    PRIMARY KEY (tenant_id, identifier)
 );
 
 -- This table some local metadata about identities
 CREATE TABLE named_identity
 (
-    tenant_id  TEXT NOT NULL,        -- Tenant identifier
-    identifier TEXT NOT NULL UNIQUE, -- Identity identifier
-    name       TEXT UNIQUE,          -- user-specified name
-    vault_name TEXT NOT NULL,        -- name of the vault used to store the identity keys
-    is_default BOOLEAN DEFAULT FALSE -- boolean indicating if this identity is the default one
+    tenant_id  TEXT NOT NULL,         -- Tenant identifier
+    identifier TEXT NOT NULL,         -- Identity identifier
+    name       TEXT NOT NULL,         -- user-specified name
+    vault_name TEXT NOT NULL,         -- name of the vault used to store the identity keys
+    is_default BOOLEAN DEFAULT FALSE, -- boolean indicating if this identity is the default one
+    PRIMARY KEY (tenant_id, identifier)
 );
 
+CREATE UNIQUE INDEX named_identity_index ON named_identity (tenant_id, name);
 
 -- This table lists attributes associated to a given identity
 CREATE TABLE identity_attributes
 (
     tenant_id   TEXT    NOT NULL, -- Tenant identifier
-    identifier  TEXT PRIMARY KEY, -- identity possessing those attributes
+    identifier  TEXT    NOT NULL, -- identity possessing those attributes
     attributes  BYTEA   NOT NULL, -- serialized list of attribute names and values for the identity
     added       INTEGER NOT NULL, -- UNIX timestamp in seconds: when those attributes were inserted in the database
     expires     INTEGER,          -- optional UNIX timestamp in seconds: when those attributes expire
     attested_by TEXT,             -- optional identifier which attested of these attributes
-    node_name   TEXT    NOT NULL  -- node name to isolate attributes that each node knows
+    node_name   TEXT    NOT NULL, -- node name to isolate attributes that each node knows
+    PRIMARY KEY (tenant_id, identifier, node_name)
 );
 
-CREATE UNIQUE INDEX identity_attributes_index ON identity_attributes (identifier, node_name);
+CREATE UNIQUE INDEX identity_attributes_index ON identity_attributes (tenant_id, identifier, node_name);
```

Comment:
there is a unique index already on these  created implicitly by the ` primary key ()` definition

Reply:
Thanks, I indeed forgot to remove that one.